web/README.md

```diff
@@ -0,0 +1,3 @@
+## Important Links
+
+- [Routing](https://www.pluralsight.com/guides/how-to-set-react-router-default-route-redirect-to-home)
```

Comment:
Probably change this to "Getting Started with React Routing"

Reply:
Updated